2021-09-06 22:13:28,719 INFO    MainThread:18208 [wandb_setup.py:_flush():69] setting env: {}
2021-09-06 22:13:28,720 INFO    MainThread:18208 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-06 22:13:28,720 INFO    MainThread:18208 [wandb_init.py:_log_setup():342] Logging user logs to D:\EPFL\pinet\wandb\run-20210906_221328-3drncjs5\logs\debug.log
2021-09-06 22:13:28,720 INFO    MainThread:18208 [wandb_init.py:_log_setup():343] Logging internal logs to D:\EPFL\pinet\wandb\run-20210906_221328-3drncjs5\logs\debug-internal.log
2021-09-06 22:13:28,720 INFO    MainThread:18208 [wandb_init.py:init():375] calling init triggers
2021-09-06 22:13:28,720 INFO    MainThread:18208 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-09-06 22:13:28,720 INFO    MainThread:18208 [wandb_init.py:init():424] starting backend
2021-09-06 22:13:28,720 INFO    MainThread:18208 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn, using: spawn
2021-09-06 22:13:28,723 INFO    MainThread:18208 [backend.py:ensure_launched():135] starting backend process...
2021-09-06 22:13:28,819 INFO    MainThread:18208 [backend.py:ensure_launched():139] started backend process with pid: 16296
2021-09-06 22:13:28,820 INFO    MainThread:18208 [wandb_init.py:init():429] backend started and connected
2021-09-06 22:13:28,821 INFO    MainThread:18208 [wandb_init.py:init():477] updated telemetry
2021-09-06 22:13:28,822 INFO    MainThread:18208 [wandb_init.py:init():500] communicating current version
2021-09-06 22:13:31,213 INFO    MainThread:18208 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.1 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-06 22:13:31,215 INFO    MainThread:18208 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-09-06 22:13:31,461 INFO    MainThread:18208 [wandb_init.py:init():540] starting run threads in backend
2021-09-06 22:13:34,853 INFO    MainThread:18208 [wandb_run.py:_console_start():1584] atexit reg
2021-09-06 22:13:34,854 INFO    MainThread:18208 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-09-06 22:13:34,855 INFO    MainThread:18208 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-09-06 22:13:34,858 INFO    MainThread:18208 [wandb_run.py:_redirect():1519] Redirects installed.
2021-09-06 22:13:34,858 INFO    MainThread:18208 [wandb_init.py:init():565] run started, returning control to user process
2021-09-06 22:13:35,918 INFO    MainThread:18208 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 1
2021-09-06 22:13:35,918 INFO    MainThread:18208 [wandb_run.py:_restore():1526] restore
2021-09-06 22:13:36,479 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 513
}

2021-09-06 22:13:36,856 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 513
}

2021-09-06 22:13:37,554 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 8051
}

2021-09-06 22:13:37,663 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 8077
}

2021-09-06 22:13:37,773 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 513
  total_bytes: 8077
}

2021-09-06 22:13:37,879 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8077
  total_bytes: 8077
}

2021-09-06 22:13:37,987 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8077
  total_bytes: 8077
}

2021-09-06 22:13:38,094 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8077
  total_bytes: 8077
}

2021-09-06 22:13:38,198 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8077
  total_bytes: 8077
}

2021-09-06 22:13:38,303 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8077
  total_bytes: 8077
}

2021-09-06 22:13:38,669 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8077
  total_bytes: 8077
}

2021-09-06 22:13:38,784 INFO    MainThread:18208 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 8077
  total_bytes: 8077
}

2021-09-06 22:13:40,019 INFO    MainThread:18208 [wandb_run.py:_show_files():1898] logging synced files
